includes/discovery/loadbalancers/f5-ltm.inc.php

```diff
@@ -44,38 +44,56 @@
 // Begin our master array, all other values will be processed into this array.
 $tblBigIP = array();
 
-// Let's gather some data..
 // Virtual Server Data
-$ltmVirtualServEntry[name] = snmpwalk_array_num($device, '1.3.6.1.4.1.3375.2.2.10.1.2.1.1', 0);
-$ltmVirtualServEntry[ip] = snmpwalk_array_num($device, '1.3.6.1.4.1.3375.2.2.10.1.2.1.3', 0);
-$ltmVirtualServEntry[port] = snmpwalk_array_num($device, '1.3.6.1.4.1.3375.2.2.10.1.2.1.6', 0);
-$ltmVirtualServEntry[defaultpool] = snmpwalk_array_num($device, '1.3.6.1.4.1.3375.2.2.10.1.2.1.19', 0);
-$ltmVsStatusEntry[state] = snmpwalk_array_num($device, '1.3.6.1.4.1.3375.2.2.10.13.2.1.2', 0);
-$ltmVsStatusEntry[errorcode] = snmpwalk_array_num($device, '1.3.6.1.4.1.3375.2.2.10.13.2.1.5', 0);
+$ltmVirtualServOID = array(
+    name => '1.3.6.1.4.1.3375.2.2.10.1.2.1.1',
+    ip => '1.3.6.1.4.1.3375.2.2.10.1.2.1.3',
+    port => '1.3.6.1.4.1.3375.2.2.10.1.2.1.6',
+    defaultpool => '1.3.6.1.4.1.3375.2.2.10.1.2.1.19',
+    state => '1.3.6.1.4.1.3375.2.2.10.13.2.1.2',
+    errorcode => '1.3.6.1.4.1.3375.2.2.10.13.2.1.5',
+);
+
+foreach ($ltmVirtualServOID as $key => $value) {
+    $ltmVirtualServEntry[$key] = snmpwalk_array_num($device, $value, 0);
```

Comment:
I would define `$ltmVirtualServEntry = [];` before the foreach loop (makes sure you don't have variable collisions)

Then you would also need to change the !is_null() calls to !empty() calls.